config/offices.yml

```diff
@@ -2,7 +2,7 @@
 defaults: &defaults
   # Which office to send the fax to in the event the submitted
   # zip code is not in our mapping
-  fallback_office: "union"
+  fallback_office: "clio"
```

Comment:
good change! the different receiving office fallback in different envs was confusing to me when I was working on the `office_location` stuff